docs/deployment.md

```diff
@@ -0,0 +1,159 @@
+# Deployment guide
+
+This document covers deployment-related actions in rif-rollup. The deployment script provides multiple utilities to
+prepare and deploy the different rif-rollup components.
+
+The deployment script can take advantage of docker and reduce the amount of components to build.
+
+## Prerequisites
+
+By default the rif-rollup system will use the `dev` environment configuration; to use different environment
+configuration we need to compile the configuration folder with the command:
+
+```
+zk config compile <env_name>
+```
+
+This command looks for the folder with the same name (`<env_name>`) inside the `./etc/env/`. Once the environment is
+compiled, we can select it with the command:
+
+```
+zk env <env_name>
+```
+
+### Override
```

Comment:
I'd call it CUSTOM_ENV, or ENV_OVERRIDE, but override is tad ambiguous.

Reply:
updated